runtime/gc_modron_standard/StandardAccessBarrier.hpp

```diff
@@ -47,7 +48,9 @@ class MM_StandardAccessBarrier : public MM_ObjectAccessBarrier
 #if defined(J9VM_GC_GENERATIONAL)
 	MM_GenerationalAccessBarrierComponent _generationalAccessBarrierComponent;	/**< Generational Component of Access Barrier */
 #endif /* J9VM_GC_GENERATIONAL */
-	
+#if defined(OMR_GC_REALTIME)
+	bool _doubleBarrierActive; /**< Global indicator that the double barrier is active. New threads will be set to double barrier mode if this falg is true. */
```

Comment:
is it initialized?

Reply:
not yet, I was going to see where to initialize it once we start working on the code for doublebarrier (in the follow up change), the code is guarded by isSnapshotAtTheBeginningBarrierEnabled option so should cause any issues for the existing gc policies